transcom/mymove
# B-19686 Tertiary destination address bugfix

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19686)
## [INT PR 1](https://github.com/transcom/mymove/pull/13078)

## Summary

Bugfix for tertiary pickup address boolean changing tertiary deliver address boolean

### How to test

1. Create a move
2. Create a shipment with secondary and tertiary pickup and delivery address and save
3. Confirm the shipment has all addresses displayed
4. Edit the shipment to not have a tertiary pickup address and save
5. Confirm that clearing out the tertiary pickup address did not change the tertiary pickup address
